pom.xml

```diff
@@ -280,6 +280,7 @@
                     <configuration>
                         <autoVersionSubmodules>true</autoVersionSubmodules>
                         <preparationGoals>clean verify install</preparationGoals>
+                        <pushChanges>false</pushChanges>
```

Comment:
After thinking about the releases more. Setting pushChanges to false is safer for testing as committers can accidentally push releases for testing. Setting this back to false should be the case until we have a solid automated release plan in place.